.travis.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -e
+
+show_help() {
+  echo "Usage: $0 <command>"
+  echo "Commands: install_docker"
+}
+
+if [[ -z $1 ]]; then
+  show_help
+  exit 1
+fi
+
+case "$1" in
+  install_docker)
+
+    if [[ -z $DOCKER_VERSION ]]; then
+      echo "DOCKER_VERSION needs to be set as an environment variable"
+      exit 1
+    fi
+
+    # TODO detect which docker version is already installed and skip
+    # uninstall/reinstall if it matches $DOCKER_VERSION
```

Comment:
Though actually I guess you still want to set `DOCKER_OPTS` so maybe not `exit 0`.